packages/cfd/src/Components/cfd-mt5-demo-account-display.tsx

```diff
@@ -98,12 +98,12 @@ const CFDMT5DemoAccountDisplay = ({
     const financial_specs = React.useMemo(() => {
         const should_show_eu = (is_logged_in && is_eu) || (!is_logged_in && is_eu_country);
         if (residence === 'au') {
-            return specifications[platform as keyof TSpecifications].au_real_financial_specs;
+            return specifications[platform as 'mt5'].au_real_financial_specs;
```

Comment:
@hamid-deriv Same in this file.

Reply:
You're right. As this file is out of this card's scope I didn't change the file more than this. BTW, I changed this.